From 3e58f9c97d95785b2bcab632871a59a04050a458 Mon Sep 17 00:00:00 2001
From: Gabriel Souza Franco <gabrielfrancosouza@gmail.com>
Date: Wed, 24 Jan 2018 14:26:28 +0100
Subject: [PATCH] qt: Fix regression from #153

Define location and soname for all configurations, not just Debug.
---
 qt/cmake/AppStreamQtConfig.cmake.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/qt/cmake/AppStreamQtConfig.cmake.in b/qt/cmake/AppStreamQtConfig.cmake.in
index aa61e820..b64a4f57 100644
--- a/qt/cmake/AppStreamQtConfig.cmake.in
+++ b/qt/cmake/AppStreamQtConfig.cmake.in
@@ -32,8 +32,8 @@ add_library(AppStreamQt SHARED IMPORTED)
 set_target_properties(AppStreamQt PROPERTIES
   INTERFACE_INCLUDE_DIRECTORIES "${PACKAGE_PREFIX_DIR}/include/"
   INTERFACE_LINK_LIBRARIES "Qt5::Core"
-  IMPORTED_LOCATION_DEBUG "@LIBDIR_FULL@/libAppStreamQt.so.${AppStreamQt_VERSION}"
-  IMPORTED_SONAME_DEBUG "libAppStreamQt.${AppStreamQt_VERSION_MAJOR}"
+  IMPORTED_LOCATION "@LIBDIR_FULL@/libAppStreamQt.so.${AppStreamQt_VERSION}"
+  IMPORTED_SONAME "libAppStreamQt.${AppStreamQt_VERSION_MAJOR}"
 )
 
 ####################################################################################
